docker/generate-config.js

```diff
@@ -1,24 +1,6 @@
 const fs = require("fs");
 const path = require("path");
 
-const readConfig = () => {
-  if (!fs.existsSync(configPath)) {
-    console.warn(
-      `Pre-start: ⚠️ No existing config file found under "${configPath}".`
-    );
-    process.exit();
-  }
-  const content = fs.readFileSync(configPath, "utf-8");
-  const existing = content.match(
-    /window\.VOCASCAN_CONFIG\s*=\s*JSON.parse\(`([\s\S]*)`\)/
-  );
-  if (!existing || existing.length < 2) {
-    console.warn("Pre-start: ⚠️ Could not find existing config in config.js");
-    process.exit();
-  }
-  return JSON.parse(existing[1]);
-};
-
```

Comment:
Why have you removed this? This would add the ability to mount a config file into the container instead of using env vars :smile: - or do you think this is one step to complex?

Reply:
I thought the config.js will always be empty, but if that's not the case I will revert it. I didnt find this topic in the docs, is it missing there?

Reply:
Yes, you're right, this is missing.